galley/pkg/config/analysis/analyzers/testdata/virtualservice_host_not_found_gateway.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     istio: ingressgateway
   servers:
   - hosts:
-    - testing-01.com
+    - '*/testing-01.com'
```

Comment:
shouldn't we have a test of all different forms to make sure they make sense?

Reply:
@howardjohn Done. PTAL. Thanks!